app/components/slider/slider.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import ReactSlider from "react-slider";
+
+export type SliderProps = Omit<JSX.IntrinsicElements["input"], "type">;
+
+export function Slider({ className, ...rest }: SliderProps) {
+  return (
+    <ReactSlider className="horizontal-slider" thumbClassName="slider-thumb" trackClassName="slider-track" {...rest} />
```

Comment:
nit: forward the `className` prop through, like this:

```suggestion
    <ReactSlider className={`horizontal-slider ${className || ""}`} thumbClassName="slider-thumb" trackClassName="slider-track" {...rest} />
```

Reply:
Removed this because it's not used right now.

Reply:
If someone passes in `className` as a prop (e.g. because they want to add margins or padding, which is a pretty normal use case) then it will override the `horizontal-slider` className prop and probably break the styling, so I think it's worth forwarding through the className prop like we do for the other components under `app/components`

Reply:
Done.